cdc/processor/processor.go

```diff
@@ -109,6 +109,25 @@ func newProcessor(ctx cdcContext.Context) *processor {
 	return p
 }
 
+// processorIgnoralbeError returns true if the error means the processor exits
+// normally, caused by changefeed pause, remove, etc.
+func processorIgnoralbeError(err error) bool {
```

Comment:
may be could include err==nil case, since already extract as a func

Reply:
addressed